data/simpy.gram

```diff
@@ -324,10 +324,28 @@ power[expr_ty]:
     | primary 
 primary[expr_ty]:
     | !'await' a=atom !'.' !'(' !'[' { a }
```

Comment:
I think we don't need this one any more. It was a shortcut from the time when most primaries didn't generate an AST at all. (Though it possibly speeds up parsing?)

Reply:
Let's go with the simplified version for now and care about speed when we have a complete grammar!